pkg/agent/utils.go

```diff
@@ -51,6 +52,23 @@ var TranslatedKubeletConfigFlags map[string]bool = map[string]bool{
 	"--resolv-conf":                       true,
 	"--system-reserved":                   true,
 	"--kube-reserved":                     true,
+	"--cpu-manager-policy":                true,
+	"--cpu-cfs-quota":                     true,
+	"--cpu-cfs-quota-period":              true,
+	"--topology-manager-policy":           true,
+	"--allowed-unsafe-sysctls":            true,
+}
+
+// SysctlDefaults are some default sysctl values aks set, but only apply if not set by custom configs from API
+var SysctlDefaults map[string]string = map[string]string{
```

Comment:
maybe I missed it, where do we use this?

Reply:
Good catch. This was used by old code, removed.